lesson04/assignment/css/_components.content.scss

```diff
@@ -0,0 +1,145 @@
+//** Reference Source Material: https://www.youtube.com/watch?v=FeJEEE3zc4U **/
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+	margin: 0;
+}
+
+body {
+	height: auto;
+	display: grid;
+	place-items: center;
+	font-family: "Arvo", sans-serif;
+}
+
+.event {
+	width: 350px;
+	height: auto;
+
+	&__content {
+		position: relative;
+		padding: 15rem 5rem;
+		text-align: center;
+		transition: transform 2.5s;
+		transform-style: preserve-3d; /*We want this to rotate independent from the parent to do we go to card content and transform style  preserve 3d*/
```

Comment:
This is a great place for a comment, as this is a rule that devs might not be familiar with, but the comment itself needs a little editing for readability.